components/Introduction.js

```diff
@@ -4,7 +4,7 @@ import { Row, Col } from "react-bootstrap";
 const Introduction = () => {
 	return (
 		<Row className='font-lexend flex justify-between items-center w-11/12 mx-auto'>
-			<Col className='text-center'>
+			<Col className='text-center w-11/12'>
```

Comment:
this shouldn't be applied here, since it would make the column take 11/12th of 50% of 11/12th of the entire screen. We want it to take up 50% of 11/12th of the entire screen for these columns. Hopefully that makes sense.